receiver/mongodbatlasreceiver/testdata/events/golden/project-events.json

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
Golden files should be yaml only at this point. Looks like you may have duplicates in your PR

Reply:
removed